src/kemalyst-model.cr

```diff
@@ -101,99 +111,104 @@ class Kemalyst::Model
       {% end %}
       return params
    end
-  end # End of Fields Macro
```

Comment:
Lots of changed lines below due to continuing the macro block to include the rest of the instance methods.